src/components/singleBook.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SingleBook = (props) => {
+  const { title, author } = props;
+
+  const handleClick = () => {
+    alert('Do You really want to delete this book from the list');
```

Comment:
- [Optional] It is [best js practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#javascript-in-the-browser) not to use alert. Kindly remove this please.